TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/InitRobot.java

```diff
@@ -0,0 +1,139 @@
+package org.firstinspires.ftc.teamcode.api;
+
+import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorEx;
+import com.qualcomm.robotcore.hardware.NormalizedColorSensor;
+import com.qualcomm.robotcore.hardware.Servo;
+
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.teamcode.api.config.Constants;
+import org.firstinspires.ftc.teamcode.api.config.Naming;
+import org.firstinspires.ftc.teamcode.api.hw.SmartColorSensor;
+import org.firstinspires.ftc.teamcode.api.hw.SmartMotor;
+import org.firstinspires.ftc.teamcode.api.hw.SmartServo;
+import org.firstinspires.ftc.teamcode.api.hw.Gyroscope;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+import java.util.Objects;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * The Robot Initializer. Place initialization code here. This prevents needing to sync the init code
+ * between all OpModes.
+ */
+public class InitRobot {
```

Comment:
Shouldn't this be renamed to be relevant to the merging of the robot classes?

Reply:
So the init should be a part of the robot class specific to the robot that this class is initializing

Reply:
InitRobot is going to be deleted, replaced by Robot.initTeleOp method.